templates/stamp/mongodb-ubuntu-install.sh

```diff
@@ -146,7 +149,7 @@ install_mongodb()
     fi
 
     # Install updates
-    apt-get -y update
+    apt-get -y -qq update
```

Comment:
i've already fixed this in a pull request that's awaiting approval. i refactored all updates to a shared function, but it looks like you've already merged so i'll revert my change